tests/validators/tests.py

```diff
@@ -45,7 +45,10 @@
     (validate_email, 'email@localhost', None),
     (EmailValidator(whitelist=['localdomain']), 'email@localdomain', None),
     (validate_email, '"test@test"@example.com', None),
+    (validate_email, 'example@atm.longgtldfortesting', None),
```

Comment:
I'd suggest something like `'example@atm.%s' % 'a' * 63` (and 64 for the invalid one)